logs/parse_test.go

```diff
@@ -442,29 +442,19 @@ var parseTests = []parseTestpair{
 		},
 		true,
 	},
-	// Custom 11 format
 	{
-		logs.LogPrefixCustom11,
```

Comment:
It looks like the existing test here was removed. I assume that was accidental?

Reply:
Yes, sorry, added back 😅